From e23b3b9c1bea99ace75a57ee0d11cb6d603f24c4 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:21 -0400
Subject: [redhat] kabi: Adding symbol si_meminfo (mm/page_alloc.c)

Message-id: <e0da2d55d4fb1333a77869f021cd32000905f618.1410510773.git.poros@redhat.com>
Patchwork-id: 96126
O-Subject: [PATCH RHEL7.1 201/233] kabi: Adding symbol si_meminfo (mm/page_alloc.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    This is used to read details of memory available,
    page sizes etc., in the running system,
    this info is then used to set caps on maximum memory
    usage of internal threads so as to maintain system stability.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/si_meminfo
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/si_meminfo
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/si_meminfo

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/si_meminfo b/redhat/kabi/kabi-module/kabi_ppc64/si_meminfo
new file mode 100644
index 0000000..c9131f1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/si_meminfo
@@ -0,0 +1,2 @@
+#1-
+0x40c7247c si_meminfo vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/si_meminfo b/redhat/kabi/kabi-module/kabi_s390x/si_meminfo
new file mode 100644
index 0000000..c9131f1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/si_meminfo
@@ -0,0 +1,2 @@
+#1-
+0x40c7247c si_meminfo vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/si_meminfo b/redhat/kabi/kabi-module/kabi_x86_64/si_meminfo
new file mode 100644
index 0000000..c9131f1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/si_meminfo
@@ -0,0 +1,2 @@
+#1-
+0x40c7247c si_meminfo vmlinux EXPORT_SYMBOL
-- 
1.7.1